libldbrest/endpoints.go

```diff
@@ -90,19 +90,21 @@ func getItems(w http.ResponseWriter, r *http.Request, p httprouter.Params) {
 		return
 	}
 
-	results := make(map[string]string, len(req.Keys))
+	results := make([]*keyval, 0, len(req.Keys))
 	for _, key := range req.Keys {
 		val, err := db.Get(ro, []byte(key))
 		if err != nil {
 			failErr(w, err)
 			return
 		} else if val != nil {
-			results[key] = string(val)
+			results = append(results, &keyval{key, string(val)})
 		}
 	}
 
 	w.Header().Set("Content-Type", "application/json")
-	json.NewEncoder(w).Encode(results)
+	json.NewEncoder(w).Encode(struct {
+		Data []*keyval `json:"data"`
```

Comment:
why not reuse the same struct from `iterItems`?
you could do something like (outside of func decl):

```go
type wrapper struct {
    More bool      `json:"more,omitempty"`
    Data []*keyval `json:"data"`
}
```

where the "`omitempty`" tag will (as you probably guessed) omit the key/value pair from the json structure if the value is empty.